src/examples/zap-baseline.py

```diff
@@ -349,6 +355,9 @@ def main(argv):
       logging.debug ('Ajax Spider complete')
 
     # Wait for passive scanning to complete
+    while (datetime.now() - start).seconds < ((mins * 60) + 10):
```

Comment:
yeah, totally right, both finish at same time and the external software can't start to work, that's the reason why we wait to finish.